pandas/tseries/common.py

```diff
@@ -62,9 +62,10 @@ def maybe_to_datetimelike(data, copy=False):
 class Properties(PandasDelegate):
```

Comment:
This should actually be renamed to `class AbstractTimerelatedDelegate(PandasDelegate):` (and the rest too: `DatetimeProperties` -> `DatetimeDelegate`), as there is a `AccessorProperty` which only constructs the delegates... The same names (`Property` vs `AcessorProperty` made understanding what is happening here harder :-( )